service/src/main/kotlin/fi/espoo/evaka/reports/ServiceVoucherValueReport.kt

```diff
@@ -147,6 +172,13 @@ private fun getServiceVoucherValues(
         JOIN person child ON part.child = child.id
         JOIN daycare unit ON part.placement_unit = unit.id
         JOIN care_area area ON unit.care_area_id = area.id
+        LEFT JOIN (
+            SELECT p.child_id, dg.name, p.start_date, p.end_date
+            FROM placement p
+            JOIN daycare_group_placement dgp ON p.id = dgp.daycare_placement_id
+            JOIN daycare_group dg ON dgp.daycare_group_id = dg.id
+        ) child_group ON child.id = child_group.child_id AND daterange(decision.valid_from, decision.valid_to, '[]') && 
+                daterange(child_group.start_date, child_group.end_date, '[]')
```

Comment:
This part breaks as soon as you have two different group placements during that placement. At minimum you have to use the dates on `daycare_group_placement` and even then you get duplicate rows if the child changes groups during the decision period. You'll have to aggregate the group names somehow.